### DISCUSSION TITLE
```RichTextBlock``` spacing between title and second paragraph

### DISCUSSION THREAD

**Discussion by limeniye** [microsoft/microsoft-ui-xaml#9168] [Q&A] [answered]
<h3>Description</h3>

I wrote a simple RichTextBlock code:

```XML
    <RichTextBlock TextIndent="20">
      <Paragraph FontSize="28">
        <Bold>Is this the right control?</Bold>
        <LineBreak FontSize="14"/>
      </Paragraph>
      <Paragraph>
        <Run>Use a</Run>
        <Bold>RichTextBlock</Bold>
        <Run>
          when you need support for multiple paragraphs, multi-column or other complex text layouts, or inline UI elements like images.
        </Run>
        <LineBreak/>
      </Paragraph>
      <Paragraph>
        <Run>Use a</Run>
        <Bold>TextBlock</Bold>
        <Run>
          to display most read-only text in your app. You can use it to display single-line or multi-line text, inline hyperlinks, and text with formatting like bold, italic, or underlined. TextBlock provides a simpler content model, so it's typically easier to use, and it can provide better text rendering performance than RichTextBlock. It's preferred for most app UI text. Although you can put line breaks in the text, TextBlock is designed to display a single paragraph and doesn't support text indentation.
        </Run>
        <LineBreak/>
      </Paragraph>
      <Paragraph>
        <Run>For more info about choosing the right text control, see the</Run>
        <Hyperlink Foreground="OrangeRed" NavigateUri="https://learn.microsoft.com/en-us/windows/apps/design/controls/text-controls">
          Text controls
        </Hyperlink>
        <Run>article.</Run>
      </Paragraph>
    </RichTextBlock>
```

<h3>Current behavior</h3>

The current behavior is that after the title I have a huge spacing between title and second paragraph:

![image](https://github.com/microsoft/microsoft-ui-xaml/assets/69314237/1b9d6dd2-8fe0-44ce-844b-cbd997d0101e)

And that is quite interesting because I don't have any features for this case.

<h3>Expected behavior</h3>

I'am expecting that the spacing between first and second line should be the same as in second and third line.

**Comment by kmgallahan** [comment]
Have you tried moving your first LineBreak to the start of the 2nd paragraph?

**Reply by limeniye** [reply]
Yes, It works. But in this case I will lose my ```TextIndent="20"```.
Consequently, I have tried the following way:
```XML
    <RichTextBlock TextIndent="20">
      <Paragraph FontSize="28">
        <Bold>Is this the right control?</Bold>
      </Paragraph>
      
      <!-- LineBreak in new Paragraph -->
      <Paragraph>
        <LineBreak FontSize="14"/>
      </Paragraph>

```

But result is the same.

**Reply by kmgallahan** [reply, marked as answer]
Then try:

``` xaml
<RichTextBlock TextIndent="20">
      <Paragraph>
        <Bold FontSize="28">Is this the right control?</Bold>
        <LineBreak />
```

because:

> Setting properties on LineBreak that otherwise affect the character line height of a text element (such as [FontSize](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.documents.textelement.fontsize?view=windows-app-sdk-1.4#microsoft-ui-xaml-documents-textelement-fontsize), [FontFamily](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.documents.textelement.fontfamily?view=windows-app-sdk-1.4#microsoft-ui-xaml-documents-textelement-fontfamily)) has no effect on the text rendering and appearance. The height of a text line is inferred from the other text elements that precedes the LineBreak, and the properties of a LineBreak don't modify that line height even if set explicitly.

**Reply by limeniye** [reply]
@kmgallahan , thanks for the answer.
Indeed, it works as you said.
But is it feature or bug?

**Reply by kmgallahan** [reply]
The text I copy & pasted comes from the LineBreak API docs, so I don't know what you are asking.
